File: HomeAutomation/HomeAutomation/EnvironmentalState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HomeAutomation
{
    public class EnvironmentalState
    {
        public double Temperature { get; set; }
        public double AmbientLight { get; set; }
        public bool MotionDetected { get; set; }
    }
}

File: Homeautomation/HomeAutomation/src/EnvironmentalState.java
public class EnvironmentalState {
   public boolean motionDetected;
   public int temperature;
   public int lightLevel;
   public int powerUsage;

    public EnvironmentalState(boolean motionDetected, int temperature, int lightLevel, int powerUsage) {
        this.motionDetected = motionDetected;
        this.temperature = temperature;
        this.lightLevel = lightLevel;
        this.powerUsage = powerUsage;
    }

}
